### DISCUSSION TITLE
Thank you!!

### DISCUSSION THREAD

**Discussion by sean-skt** [gotzl/hid-fanatecff#76] [General]
Hi Gotzl

I just wanted to create a post to give a massive thanks for this driver. I know you are under no obligation to develop it but any work you do on it is hugely appreciated as it has become essential for me as a Linux user. I think as time goes by and the uptake of Linux increases due to improved support for Windows games in Linux, drivers like yours will be invaluable.
Thanks again!!

**Comment by gotzl** [comment]
Thank you very much for the the kind words!

**Comment by igghcz** [comment]
I add my thanks. I've tried CSL DD in Test Drive: Ferrari Racing Legends on both Linux and Windows. Under Linux with hid-fanatecff there is even a steering wheel jerk effect when shifting gears, which is not present under Windows with the company driver.
